contrib/babelfishpg_common/sql/upgrades/babelfishpg_upgrades.in

```diff
@@ -0,0 +1,21 @@
+/*
+ * All objects created by the included files will be created in sys
+ */
+
+\echo Use "ALTER EXTENSION ""babelfishpg_common"" UPDATE TO '3.3.0'" to load this file. \quit
```

Comment:
The echo command needs to be generic. Else, every release will need an update to this file.

Reply:
Done, Replaced (3.3.0 -> CUR_VERSION)